README.md

```diff
@@ -59,6 +60,7 @@
 
 ## Deployment Tools
 
+* [colmena](https://github.com/zhaofengli/colmena) - A simple, stateless NixOS deployment tool modeled after NixOps and morph.
```

Comment:
Since [the project](https://github.com/zhaofengli/colmena) capitalizes "Colmena", we should too. Otherwise, looks good!